pandas/core/computation/pytables.py

```diff
@@ -601,8 +601,7 @@ def __init__(self, value, converted, kind: str):
         self.kind = kind
 
     def tostring(self, encoding) -> str:
-        """ quote the string if not encoded
-            else encode and return """
+        """ quote the string if not encoded else encode and return """
```

Comment:
are the leading/trailing spaces policy?

Reply:
not according to PEP257, but that seems to popular in our codebase. There is a codecheck in pydocstyle for this, D210. This is for one-line docstrings and we seem to moving away from those, see #31162, #31462 and others.

I was wandering whether to enforce D210 yet. May be able to find a checker to find one-liners.

Reply:
I think this would fail the numpy docstring check as well were this exposed as part of the API